Error: Cannot find module 'node:util'

```
LSP-json: internal/modules/cjs/loader.js:883
LSP-json:   throw err;
LSP-json:   ^
LSP-json: 
LSP-json: Error: Cannot find module 'node:util'
LSP-json: Require stack:
LSP-json: - D:\Sublime Text\Data\Package Storage\LSP-json\14.16.1\language-server\node_modules\vscode-languageserver\lib\node\main.js
LSP-json: - D:\Sublime Text\Data\Package Storage\LSP-json\14.16.1\language-server\node_modules\vscode-languageserver\node.js
LSP-json: - D:\Sublime Text\Data\Package Storage\LSP-json\14.16.1\language-server\out\node\jsonServerMain.js
LSP-json:     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
LSP-json:     at Function.Module._load (internal/modules/cjs/loader.js:725:27)
LSP-json:     at Module.require (internal/modules/cjs/loader.js:952:19)
LSP-json:     at require (internal/modules/cjs/helpers.js:88:18)
LSP-json:     at Object.<anonymous> (D:\Sublime Text\Data\Package Storage\LSP-json\14.16.1\language-server\node_modules\vscode-languageserver\lib\node\main.js:23:21)
LSP-json:     at Module._compile (internal/modules/cjs/loader.js:1063:30)
LSP-json:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
LSP-json:     at Module.load (internal/modules/cjs/loader.js:928:32)
LSP-json:     at Function.Module._load (internal/modules/cjs/loader.js:769:14)
LSP-json:     at Module.require (internal/modules/cjs/loader.js:952:19) {
LSP-json:   code: 'MODULE_NOT_FOUND',
LSP-json:   requireStack: [
LSP-json:     'D:\\Sublime Text\\Data\\Package Storage\\LSP-json\\14.16.1\\language-server\\node_modules\\vscode-languageserver\\lib\\node\\main.js',
LSP-json:     'D:\\Sublime Text\\Data\\Package Storage\\LSP-json\\14.16.1\\language-server\\node_modules\\vscode-languageserver\\node.js',
LSP-json:     'D:\\Sublime Text\\Data\\Package Storage\\LSP-json\\14.16.1\\language-server\\out\\node\\jsonServerMain.js'
LSP-json:   ]
LSP-json: }
```

node : 14.16.1
LSP-json:v1.15.0

I tried to rollback the `vscode-languageserver` version and it worked

--- a/plugin.py
+++ b/plugin.py
@@ -31,8 +31,8 @@ class LspJSONPlugin(NpmClientHandler, StoreListener):
     _schema_store = SchemaStore()
 
     @classmethod
-    def minimum_node_version(cls) -> Tuple[int, int, int]:
-        return (14, 0, 0)
+    def required_node_version(cls) -> str:
+        return '>=18'
 
     @classmethod
     def cleanup(cls) -> None:

